contracts/crosschain/README.adoc

```diff
@@ -0,0 +1,42 @@
+= Crosschain
+
+[.readme-notice]
+NOTE: This document is better viewed at https://docs.openzeppelin.com/community-contracts/crosschain
```

Comment:
```suggestion
NOTE: This document is better viewed at https://docs.openzeppelin.com/community-contracts/api/crosschain
```

Reply:
Interesting, I just noticed most other README references are broken, I'm committing this and we can solve others in a different PR